variables.tf

```diff
@@ -0,0 +1,61 @@
+# ENVIRONMENT
+variable "confluent_environment_name" {
+  type        = string
+  description = "The name of the environament you want to use."
+}
+
+# CLUSTER
+variable "kafka_cluster_type" {
+  type        = string
+  default     = "basic"
+  description = "The configuration of the Kafka cluster."
+}
+
+variable "kafka_cluster_name" {
+  type        = string
+  description = "The name of the Kafka cluster."
+}
+
+variable "kafka_cluster_availability_zone" {
+  type        = string
+  default     = "SINGLE_ZONE"
+  description = "The availability zone configuration of the Kafka cluster. Accepted values are: SINGLE_ZONE and MULTI_ZONE."
+}
+
+variable "kafka_cluster_cloud_provider" {
+  type        = string
+  default     = "GCP"
+  description = "The cloud service provider that runs the Kafka cluster. The accepted values are: GCP, AWS and Azure."
+}
+
+variable "kafka_cluster_region" {
+  type        = string
+  description = "The configuration region of the Kafka cluster."
+}
+
+variable "dedicated_cluster_cku" {
+  type        = number
+  default     = 0
+  description = "The number of Confluent Kafka Units for Dedicated cluster types. The minimum number of CKUs for SINGLE_ZONE dedicated clusters is 1 whereas MULTI_ZONE dedicated clusters must have more than 2 CKUs."
+}
+
+# TOPICS
+variable "topics" {
+  type = map(object({
+    partitions_count = number
+    config           = map(string)
+  }))
+  default     = {}
+  description = "This helps to create a Topic resource along with the specific settings."
+}
+
+# API Keys
+variable "services" {
```

Comment:
can we rename this var to something more meaningful?
ex: services_acls, iam_services

Reply:
Variable renamed.